src/foam/core/crunch/lite/ruler/ReputDependantPayloads.js

```diff
@@ -40,7 +40,7 @@ foam.CLASS({
             CapabilityJunctionPayload payload = (CapabilityJunctionPayload) obj;
             if ( old != null && old.getStatus() == payload.getStatus() ) return;
 
-            var payloadDAO = (DAO) getX().get("capablePayloadDAO");
+            var payloadDAO = (DAO) getX().get("capabilityPayloadDAO");
```

Comment:
@noodlemoodle do you recall the original version of this? I can't find a cspec for capablePayloadDAO. 
Similar in rule below.